src/translations/index.ts

```diff
@@ -8,7 +8,8 @@
 const translations = {
   en: require('./en.json'),
   ja: require('./ja.json'),
-  'ja-Hira': require('./ja-Hira.json')
+  'ja-Hira': require('./ja-Hira.json'),
+  'zh-CN': require('./zh-CN.json')
```

Comment:
This is case-sensitive.
```suggestion
  'zh-cn': require('./zh-CN.json')
```